docs/source/agent-framework/web-framework.rst

```diff
@@ -54,7 +91,11 @@ JSON endpoints allows an agent to serve data responses to specific queries from
         self.vip.web.register_endpoint(r'/vc/jsonrpc', self.jsonrpc)
 
 
-Websocket endpoints allow bi-directional communication between the client and the server.  Client connections can be authenticated during the opening of a websocket through the response of an open callback.
+Websocket
+~~~~~~~~~
+
+Websocket endpoints allow bi-directional communication between the client and the server.
+Client connections can be authenticated during the opening of a websocket through the response of an open callback.
```

Comment:
In the open of the websockets, lets provide an example of how the user agent would protect against unlawful entry using the minimum requirements.

Note that there is one more aspect of this that needs to be rewritten, however I have to figure out how to serialize a Request and JsonRequest through the vip message bus.  So that agents can use those objects to return.